requirements.txt

```diff
@@ -0,0 +1,13 @@
+Django
+Jinja2
+MarkupSafe
+PyYAML
+Pygments
+Sphinx
+argparse
+docutils
+mock
+pep8
+psycopg2
+requests
+wsgiref
```

Comment:
@Kennric @athai I was incorrect in assuming we needed to duplicate the requirements file. I just pushed an update that seems to build fine without the requirements file in the `dockerfiles` directory.